plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -5474,6 +5517,14 @@ def main():
         ccc_events.check_return_status()
 
     ccc_events.validate_input().check_return_status()
+    if ccc_events.get_ccc_version_as_integer() <= ccc_events.get_ccc_version_as_int_from_str("2.3.5.3"):
```

Comment:
if ccc_events.compare_dnac_versions(ccc_sda_devices.get_ccc_version(), "2.3.5.3") < 0:
        ccc_events.msg = (
            "The specified version '{0}' does not support the events and notifications workflow. Supported versions start from '2.3.5.3' onwards."
            .format(ccc_events.get_ccc_version())
        )
        ccc_events.set_operation_result("failed", False, ccc_events.msg, "ERROR").check_return_status()

Reply:
updated.